### ISSUE TITLE
Livestream Interaction: Like, Save, Share, Donate

### ISSUE THREAD

**Issue by JasonHighVibe** [HighVibe/hvn#40]
Objective:
Users will be able to like, save, share, and donate directly to Creators.

Liking will show support to the Author (and give the person liking some XP).
Save will save the Livestream to the Users's Playlists. 
Sharing will allow users to share the livestream to others via Whatsapp, Facebook, "Copylink" - Users will receive XP for doing so.

Purpose: 
To reward users for interacting with the Livestream via sharing and supporting the Creators.
It is important to reduce the number of barriers between Creators and Users.

**Comment by 7flash** [comment]
Right, we also have to enable access to live streaming via browser.

**Comment by JasonHighVibe** [comment]
Yes #31
